### PR TITLE
Added escape char before some underscores to fix MD rendering <eom>

### PR DESCRIPTION
**Pull request by christianpaquin**

### CODE REVIEW

File: draft-bbs-signatures.md [hunk 1]
```diff
@@ -428,7 +428,7 @@ Procedure:
 
 6. for element in (e, s) do
 
-7.      element = OS2IP(h.read(xof_no_of_bytes)) mod q
+7.      element = OS2IP(h.read(xof\_no\_of\_bytes)) mod q
```
Comment (tplooker) [inline draft-bbs-signatures.md:431]: I dont think these need to be escaped do they? just the ones outside of a code block?

Comment (christianpaquin) [reply draft-bbs-signatures.md]: Good catch. Fixed (here and in CreateGenerators)

**Review by tplooker — APPROVED** [review]
LGTM

**Review by andrewwhitehead — APPROVED** [review]

**Comment by tplooker** [comment]
Minor editorial changes, multiple approvals, merging

**Pull request merged**